src/rs.cpp

```diff
@@ -2733,3 +2735,20 @@ void rs2_set_calibration_table(const rs2_device* device, const void* calibration
     auto_calib->set_calibration_table(buffer);
 }
 HANDLE_EXCEPTIONS_AND_RETURN(,calibration, device)
+
+rs2_raw_data_buffer* rs2_serialize_json(rs2_device* dev, rs2_error** error) BEGIN_API_CALL
+{
+    VALIDATE_NOT_NULL(dev);
+    auto serializable = VALIDATE_INTERFACE(dev->device, librealsense::serializable_interface);
+    return new rs2_raw_data_buffer{ serializable->serialize_json() };
+}
+HANDLE_EXCEPTIONS_AND_RETURN(nullptr, dev)
+
+void rs2_load_json(rs2_device* dev, const void* json_content, unsigned content_size, rs2_error** error) BEGIN_API_CALL
+{
+    VALIDATE_NOT_NULL(dev);
+    VALIDATE_NOT_NULL(json_content);
+    auto serializable = VALIDATE_INTERFACE(dev->device, librealsense::serializable_interface);
+    serializable->load_json(std::string(static_cast<const char*>(json_content), content_size));
```

Comment:
Any reason we're passing the json_content as 'void*' when we can just make it 'const char*'?